src/style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+  --moyo-main-color: #00acdc;
+  --text-main-color: #060b35;
+  --text-secondary-color: #616070;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
```

Comment:
check this checklist point
<img width="509" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/b98fff6a-6ad3-4997-bc88-4c769654adfc">